docs/reference/ml/trained-models/apis/put-trained-models.asciidoc

```diff
@@ -684,6 +687,115 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-config-nlp-tokenizati
 ======
 =====
 
+`question_answering`:::
+(Optional, object)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-config-nlp-question-answering]
++
+.Properties of question_answering inference
+[%collapsible%open]
+=====
+`max_answer_length`::::
+(Optional, integer)
+The maximum amount of words that the answer consists. Defaults to `15`.
+
+`question`::::
+(Required, string)
+The question that you want to be answered based on the context of the provided 
+text.
```

Comment:
`question` is not settable on PUT. This makes question_answering unique. `question` is only settable at inference time.